composer.json

```
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": "^7.1|^8.0",
-        "symfony/form": "^4.4|^5.0",
+        "symfony/form": "^4.4|^5.0|6.0",
```

This notation cannot work, because it points to a specific version (that do not exists), we must use `^6.0`

```suggestion
        "symfony/form": "^4.4|^5.0|^6.0",
```